hack/build_rpms.sh

```
@@ -8,15 +8,13 @@ if [ -n "${FLIGHTCTL_RPM:-}" ]; then
 fi
 
 # our RPM build process works in rpm bases systems so we wrap it if necessary
-if ! command -v packit 2>&1 >/dev/null ]; then
-    echo "Building RPMs on a system without packtit, using container"
+if ! command -v packit >/dev/null  2>&1 ]; then
```

Shellcheck (0.10.0)

[warning] Found trailing ] outside test. Add missing [ or quote if intentional.

(SC2171)